pyproject.toml

```diff
@@ -35,9 +36,12 @@ dependencies = [
   "Cython",
   "geomdl",
   "numpy",
+  "numpy<2",
```

Comment:
I think this is not wanted?

Reply:
No, definitely not! That would have been a short switch to numpy2 😂